kernel/environ.mli

```diff
@@ -80,6 +80,8 @@ type env = private {
   env_nb_rel        : int;
   env_universes : UGraph.t;
   env_universes_lbound : UGraph.Bound.t;
+  irr_constants : Cset.t;
```

Comment:
Wait a sec. @SkySkimmer when did you switch from `Cset_env` to `Cset`? We shouldn't use the latter in the kernel (nor anywhere if we ever want to get rid of the canonical mess).

Reply:
Isn't it fine? The relevance doesn't differ between the different aliases of a constant.

Reply:
Yes, but we shouldn't rely on the canonical-user quotient in the kernel for future-proof reasons. It doesn't hurt to use the `_env` variant, so I'll fix this.